metadata_backend/helpers/parser.py

```diff
@@ -40,6 +41,63 @@ def __init__(self,
             namespaces, dict_class, list_class, **kwargs
         )
 
+    def _to_camel(self, name: str) -> str:
+        """Convert underscore char notation to CamelCase."""
+        _under_regex = re.compile(r'_([a-z])')
+        return _under_regex.sub(lambda x: x.group(1).upper(), name)
+
+    def _flatten(self, data: Any) -> Union[Dict, List, str, None]:
+        links = ['studyLinks', 'sampleLinks', 'runlinks',
+                 'experimentLinks', 'analysisLinks', 'projectLinks',
+                 'policyLinks', 'dacLinks', 'datasetLinks',
+                 'assemblyLinks', 'submissionLinks']
+
+        children = self.dict()
+        for key, value, _ in self.map_content(data.content):
+            key = self._to_camel(key.lower())
+
+            # we flatten the attributes structure into an list of dict
+            if "Attributes" in key and len(value) == 1:
```

Comment:
For less erroneus matching, maybe use same strategy here than for links? 
```
atrributes = ['studyAttributes', 'sampleAttributes', 'runAttributes',
                 'experimentAttributes', 'analysisAttributes', 'projectAttributes',
                 'policyAttributes', 'dacAttributes', 'datasetAttributes',
                 'assemblyAttributes', 'submissionAttributes']
if key in attributes:...
```
etc.

Reply:
done.